vm/src/function.rs

```diff
@@ -549,6 +549,36 @@ macro_rules! into_py_native_func_tuple {
                 })
             }
         }
+
+        impl<F, $($T,)* R> IntoPyNativeFunc<($(OwnedParam<$T>,)*), R, ()> for F
```

Comment:
I'm still struggeling to understand this part of the code. I'm unable to review this properly...

Reply:
Would you tell me which part made the confusion? the last `()` part?